### DISCUSSION TITLE
Radicale integration in Mail-in-a-Box

### DISCUSSION THREAD

**Discussion by cmonty14** [Kozea/Radicale#1371] [Q&A general]
Hi,
I'm a fan of simple and thin software solutions.
Searching a thin solution for Cal/CardDav and I think _Radicale_ is a perfect match.
Searching a simple solution for mail server hosting [_Mail-in-a-Box_](https://mailinabox.email/) is a perfect match.

Now I wonder if these two perfect solutions could be combined?

There's a [discussion](https://github.com/mail-in-a-box/mailinabox/issues/2164) in Mail-in-a-Box about this topic, but nobody has started integration of _Radicale_, yet.

I'm not a developer and can hardly support this integration.
But as an end-user I would be happy to test it.

Ideally developers of _Radicale_ and _Mail-in-a-Box_ would start collaboration on this topic in order to replace Mail-in-a-Box's currently used Cal/CardDav service _Nextcloud_. 

THX

**Comment by return42** [comment]
> Searching a simple solution for mail server hosting [Mail-in-a-Box](https://mailinabox.email/) is a perfect match.

Sure? .. seems to be limited to [ubuntu](https://github.com/mail-in-a-box/mailinabox#in-the-box):

> Mail-in-a-Box turns a fresh Ubuntu 22.04 LTS 64-bit machine into a working mail server by installing and configuring various components.

---

> I'm not a developer and can hardly support this integration.
> Ideally developers of Radicale and Mail-in-a-Box would start collaboration on this topic in order to replace Mail-in-a-Box's currently used Cal/CardDav service Nextcloud.

I understand your intention  but from my developer's point of view both services (mail & dav) stand for its own .. integration of a DAV service into a mail server is more a thing that needs to be implemented on the mail server .. not sure if Radicale developers can really be a part of this task  / there is no need to touch Radicale to integrate (DAV) into Mail-in-a-Box.

**Comment by cmonty14** [comment]
Mail-in-a-Box limitation to Ubuntu is true, but any VPS provider offers Ubuntu images.
Certainly a mailserver would benefit from Radicale integration to offer Cal/CardDav services. On the other hand Radical's publicity would increase with well-known Mail-in-a-Box solution.

**Comment by jonassmedegaard** [comment]
Radicale is a code project: It *contains* functionality implementing server parts of CalDAV and CardDAV standards.

Mail-in-a-Box is an integration project: It *integrates* functionality contained in a range of other projects.

It is sensible for an integration project to widen its scope to cover more: The task mainly involves adapting more configuration files, to align the integrated parts to fit each other.

It is less sensible for a code project to widen its scope to cover more: The task mainly involves inventing new and fundamentally different functionality.
